src/main/scala/io/github/sentenza/hacktoberfest18/sort/CommonSortings.scala

```diff
@@ -25,11 +26,51 @@ object CommonSortings {
   }
 
   object CommonSorting {
-    def bubbleSort(array: Array[Int]): Array[Int] = ???
-    def selectionSort(array: Array[Int]): Array[Int] = ???
-    def insertionSort(array: Array[Int]): Array[Int] = ???
-    def quickSort(array: Array[Int]): Array[Int] = ???
-    def mergeSort(array: Array[Int]): Array[Int] = ???
+    def swap(array: Array[Int], index1: Int, index2: Int) = {
```

Comment:
Could you please add a docblock describing what the `swap()` function does? Do we need it to be publicly accessible?

Reply:
Hm, you are right, we do not need it here. I'll remove it from trait and will describe, what it does